app.js

```diff
@@ -1,85 +1,79 @@
-var express = require('express');
-var firebase = require('firebase');
-var config = require('./config.json');
-
-var fetch = require('isomorphic-fetch');
+const express = require('express');
+const firebase = require('firebase');
+const fetch = require('isomorphic-fetch');
+const config = require('./config.json');
 
 // Constants
 const PORT = 8080;
-const productionUrl = 'https://www.zooniverse.org/api';
-const stagingUrl = 'https://panoptes-staging.zooniverse.org/api';
-const API_URL = (process.env.NODE_ENV === 'production') ? productionUrl : stagingUrl;
+const PRODUCTION_URL = 'https://www.zooniverse.org/api';
```

Comment:
Constant primitives like strings. Have a look in the `panoptes-javascript-client` config file.